packages/manager/src/features/Databases/DatabaseDetail/index.tsx

```diff
@@ -99,7 +99,7 @@ export const DatabaseDetail = () => {
   }
 
   const isDefault = database.platform === 'rdbms-default';
-  const isMonitorEnabled = isDefault && isDatabasesMonitorEnabled;
+  const isMonitorEnabled = isDatabasesMonitorEnabled;
```

Comment:
How do we make sure this change will not go to develop branch?

Reply:
because we are not syncing any code from our branch to upstream develop, so no worries